src/CMakeLists.txt

```diff
@@ -91,12 +93,11 @@ ADD_LIBRARY(forge SHARED
     )
 
 TARGET_LINK_LIBRARIES(forge
-                      ${GLFW_LIBRARY}
-                      ${GLEWmx_LIBRARY}
-                      ${OPENGL_gl_LIBRARY}
-                      ${OPENGL_glu_LIBRARY}
-                      ${FREETYPE_LIBRARIES}
-                      ${FONTCONFIG_LIBRARY}
-                      )
+    ${FONTCONFIG_LIBRARY}
+    ${FREETYPE_LIBRARIES}
+    ${OPENGL_gl_LIBRARY}
```

Comment:
Removed gluErrorString from the code and we are not using glu anywhere apart from that one. We can write our own error string function based on error codes.